apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/ens.rb

```diff
@@ -3,6 +3,8 @@
 module ActionTracking
   module Strategies
     class Ens < Base
+      include Import["services.ens"]
```

Comment:
i have never seen this before tbh

Reply:
this is dependency injection 😃 we're doing it to avoid a dependency of the code in the domain from the "infra" code